presto-main/src/main/java/com/facebook/presto/server/ServerInfoResource.java

```diff
@@ -92,4 +94,17 @@ public NodeState getServerState()
             return ACTIVE;
         }
     }
+
+    @GET
+    @Path("coordinator")
+    @Produces(TEXT_PLAIN)
+    public Response getServerCoordinator()
```

Comment:
I am putting all the presto nodes behind a load balancer with an Extended Content Verification (ECV) content check of `GET /v1/info/coordinator`.  Because the method returns 503 for non-coordinator nodes all the traffic is routed to the coordinator.